src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -2199,14 +2198,13 @@ def check_cl_registration_and_get_oid(cmd, cl_oid, subscription_id):
 
 def get_custom_locations_oid(cmd, cl_oid):
     try:
-        sp_graph_client = get_graph_client_service_principals(cmd.cli_ctx)
-        sub_filters = []
-        sub_filters.append("appId eq '{}'".format("bc313c14-388c-4e7d-a58e-70017303ee3b"))  # Lookup using well-known first-party application id
-        result = list(sp_graph_client.list(filter=(' and '.join(sub_filters))))
-        if len(result) != 0:
-            if cl_oid is not None and cl_oid != result[0].object_id:
-                logger.debug("The 'Custom-locations' OID passed is different from the actual OID({}) of the Custom Locations RP app. Proceeding with the correct one...".format(result[0].object_id))
-            return result[0].object_id  # Using the fetched OID
+        graph_client = graph_client_factory(cmd.cli_ctx)
```

Comment:
CLI is already making use of MS graph in other places, you are just reusing the same here . Is that correct? @TheOnlyWei

Reply:
I am using the Azure CLI MS Graph client that they use for their RBAC module.